src/dicg.jl

```diff
@@ -491,52 +515,86 @@ function lazy_dicg_step(
     phi,
     epsilon,
     d;
-    use_extra_vertex_storage=false,
-    extra_vertex_storage=nothing,
-    lazy_tolerance=2.0,
-    memory_mode::MemoryEmphasis=InplaceEmphasis(),
+    variant = "standard",
```

Comment:
Since you only have two variants it might be cleaner to use a boolean flag here. For example, `using_blended` which would be true if blended_dicg was used and false otherwise.